nextflow_schema.json

```diff
@@ -630,5 +630,343 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
-}
+    ],
+    "properties": {
```

Comment:
None of these parameters are grouped. Please organise them into groups in the schema with logical ordering.

Ungrouped schema parameters will soon lead to a failure with nf-core lint.